plugin.xml

```diff
@@ -61,7 +61,7 @@
       <preference name="GradlePluginGoogleServicesVersion" value="4.3.8" />
 
       <preference name="GradlePluginKotlinEnabled" value="true" />
-      <preference name="GradlePluginKotlinVersion" value="1.5.20" />
+      <preference name="GradlePluginKotlinVersion" value="1.7.10" />
```

Comment:
```suggestion
      <preference name="GradlePluginKotlinVersion" value="1.7.21" />
```

Can we bump this to `1.7.21` to match with Cordova-Android 12?

Since there is other tickets to support Android 13 and it uses permissions that were added in API 33, it means this plugin will need to set the minimum requirement for `cordova-android` to `>=12.0.0`..

To match the minimum requirement.. I think it would be better to also set the default `GradlePluginKotlinVersion` to the same version as `12.0.0`